app/controllers/message_drafts_controller.rb

```diff
@@ -44,7 +44,7 @@ def create
       @templates_list = MessageTemplate.tenant_templates_list(Current.tenant)
       @message_template ||= MessageTemplate.default_template
       @boxes = Current.tenant.boxes.pluck(:name, :id)
-      @box = @box&.slice(:name, :id).values.to_a
+      @box = @box&.slice(:name, :id)&.values.to_a
```

Comment:
Tieto & "train wreck" mi smrdia. Preco vlastne z boxu robime array?

Reply:
Kvoli vyuzitiu select elementu, kde zobrazujeme nazov boxu, ale chceme aby value bola `id` boxu, https://github.com/slovensko-digital/govbox-pro/blob/main/app/components/new_message_draft_component.html.erb#L18

Reply:
Tej casti rozumiem ale @box je zjavne typu Box a odrazu je z neho array.

Reply:
@luciajanikova tuto sa mi nepozdava stale to, ze mame @box kde by som ocakaval, ze to bude box a nie nejaky divny hash/array. Asi to premenujme alebo nieco

Reply:
Upravila som, pozri prosim @jsuchal

Reply:
@luciajanikova inak mozno este naopak, ze preco tam vlastne neposleme selected box naozaj a tieto imp. detaily neriesime nizsie?